readers-frontend/src/components/SolidJS/SearchArticle.tsx

```diff
@@ -0,0 +1,91 @@
+import type { Component } from "solid-js";
+
+interface SearchArticleProps {
+  headline: string;
+  excerpt: string | null;
+  author: string;
+  authorId: number | undefined;
+  category: string | null | undefined;
+  categoryLink: string | undefined;
+  categoryColor: string | undefined;
+  imageUrl: string | false;
+  articleUrl: string;
+  isVertical?: boolean; // Defines if the image is ontop or to the left of the text
+  isPortrait: boolean; // Defines if the image is portrait or landscape
+  hideCategoryAccent?: boolean;
+  textColour?: string;
+}
+
+const SearchArticle: Component<SearchArticleProps> = (props) => {
+  console.log(
```

Comment:
This is polluting the terminal output btw. We should just add it as a comment instead

Reply:
I disagree. We're going to have a ticket about it later where we will go through the TODOs. If you want to make it clearer, we can add them to the issues tab here on github.